ocaml/typing/jkind.mli

```diff
@@ -70,7 +70,9 @@ end
     [Any] and subjkinds of other sorts, such as [Immediate]. *)
 type t = Types.type_expr Jkind_types.t
 
-include module type of Jkind_intf.History
+include module type of struct
+  include Jkind_intf.History
+end
```

Comment:
Is there a semantice difference between
```
include module type of Jkind_intf.History
```
and
```
include module type of struct
  include Jkind_intf.History
end
```
?

Reply:
Sadly, yes.

```
# module M = struct type t end;;
module M : sig type t end
# module type S1 = module type of M;;
module type S1 = sig type t end
# module type S2 = module type of struct include M end;;
module type S2 = sig type t = M.t end
```

Note that `S2` remembers that its type `t` is the same as `M.t`, necessary for our use case.